build.xml

```diff
@@ -1035,4 +1039,11 @@
       </classpath>
     </eclipse>
   </target>
+
+  <target name="check-forbidden-apis" depends="compile">
+    <fa:forbiddenapis internalRuntimeForbidden="true" classpathref="classpath" dir="${build.dir}">
+      <bundledsignatures name="jdk-unsafe-${javac.version}"/>
```

Comment:
I've prepared much more complicated PR for this. However I got the same error. So, I wanted to solve it and then improve the PR. Do you have any idea about the error?